DQMServices/Components/scripts/dxr-index.cpp

```diff
@@ -0,0 +1,1490 @@
+#include "clang/AST/AST.h"
```

Comment:
Having source code in `scripts` looks weird.

Reply:
Similar -- I just wanted some place to put the files under version control, but clearly this should not be compiled into CMSSW. Also, it does not really fit bin/, since actually that compiles into a library.

Getting access to the DXR index database in some official way might be useful, but for now, this pile of scripts has served me fairly well... (Afair scram can build the dxr index, but actually I didn't even bother to check what the output of that would be).

Reply:
I put the tools onto a different branch for now. I'd still be happy about proposals for a more permanent place for such things.

Reply:
I'm pretty sure someone will object, but `test`? (technically these are not unit tests, but we tend to have "random stuff" in `test` in many packages)